cypress/integration/manualTests/monitor.ts

```diff
@@ -50,3 +52,27 @@ describe('Monitor homepage links', () => {
     });
   });
 });
+
+describe('Test launch button icons', () => {
+  it('find a WDL workflow', () => {
+    cy.visit('/search');
+    cy.wait(10000); // wait for the tabs to become clickable
+    goToTab('Workflows');
+
+    // click to sort by descriptor type descending so WDL is at the top
+    cy.get('[data-cy=descriptorType]')
+      .click()
+      .click();
+    cy.get('[data-cy=workflowColumn]')
+      .first()
+      .within(() => {
+        cy.get('a').click(); // click on the link to the first workflow
+      });
+  });
+
+  it('get the svg icons', () => {
+    cy.get('[data-cy=dnanexusIcon]').should('exist');
```

Comment:
Have you tested if this catches the case when the icon is not present?

You can make the network request fail locally by blocking it.

I'm concerned this element may be present if the request to get the icon fails; successfully fetching the icon appends SVG DOM elements, which I think you may need to look for.

Reply:
You're right, should've caught that. Changed it to check for a svg element